src/Exception/Database/DatabaseExceptionTrait.php

```diff
@@ -7,7 +7,7 @@
     /** @var string */
     protected $driver;
 
-    private static $driverNames = [
+    protected static $driverNames = [
```

Comment:
Hm that is strange. When I change the name I still get this error. Could it be a PHP version thing? I'm using 5.6.20.